src/main/java/list_Entry.java

```diff
@@ -0,0 +1,42 @@
+public class list_Entry {
```

Comment:
May have to change the class name to fit Java class naming standards? Eg, ListEntry